addOns/help/src/main/javahelp/contents/start/features/authmethods.html

```diff
@@ -0,0 +1,160 @@
+			href="../../ui/dialogs/session/context-auth.html">Session
+			Contexts Authentication screen</a> or using the contextual PopupMenu: <i>Flag as...
+			Form-Based Authentication Login Request</i>.
+	</p>
+	<p>When using this authentication method, configuring a User for the context requires
+	setting up the <i>username/password</i> pair of credentials that are used for the form based
+	authentication.</p>
+	<p>
+	If the application requires submitting the anti-CSRF token presented in the login page, ZAP will 
+	handle it automatically. Make sure the token
+	name is configured in <a href="../../ui/dialogs/options/anticsrf.html">Options Anti CSRF screen</a>.
+	</p>
+	
+	<h2>
+		<a name="jsonBased">JSON-Based Authentication</a>
+	</h2>
+	<p>
+		This method is used for websites / webapps where authentication is
+		done by submitting a JSON object to a 'login url' using a 'username/password'
+		pair of authentication credentials.
+		Re-authentication is possible. Configuration can be done using the <a
+			href="../../ui/dialogs/session/context-auth.html">Session
+			Contexts Authentication screen</a> or using the contextual PopupMenu: <i>Flag as...
+			JSON-Based Authentication Login Request</i>.
+	</p>
+	<p>When using this authentication method, configuring a User for the context requires
+	setting up the <i>username/password</i> pair of credentials that are used for the
+	authentication.</p>
+	
+	<h2>
+		<a name="httpAuth">HTTP/NTLM Authentication</a>
+	</h2>
+	<p>
+		This method is used for websites / webapps where authentication is
+		enforced using the HTTP or NTLM Authentication mechanisms employing HTTP message headers. 
+		Three authentication schemes are supported: Basic, Digest and NTLM.
+		Re-authentication is possible, as the authentication headers are sent with every authenticated
+		request. Configuration can be done using the <a
+			href="../../ui/dialogs/session/context-auth.html">Session
+			Contexts Authentication screen</a>.
+	</p>
+	<p>When using this authentication method, configuring a User for the context requires
+	setting up the <i>username/password</i> pair of credentials that are used for the HTTP/NTLM authentication.</p>
+	
+	<h2>
+		<a name="scriptBased">Script-Based Authentication</a>
+	</h2>
+	<p>
+		This method is useful for websites / webapps where the authentication is a more complex one and some custom
+		scripts that handle the authentication process are beneficial. To use this method, you must first define an Authentication script
+		which sends messages or performs other actions as needed by your web-application. This script is then selected for use for a given Context
+		and it is called whenever an authentication is performed. Re-authentication is possible. 
+		Configuration can be done using the <a href="../../ui/dialogs/session/context-auth.html">Session
+			Contexts Authentication screen</a> and requires you to have the Scripts Console ZAP Addon installed from the Marketplace.
```

Comment:
- Remove "ZAP" that's implied?
 - `Addon` → `add-on`?
 - Technically the add-on is not required one can still select the scripts without it. Recommend instead?